### PR TITLE
Add PyBay 2019 conference

### PR DESCRIPTION
**Pull request by CloudChaoszero**
Hello team!
Was hoping if you could add PyBay 2019 Conference to the Python Conference listings!

**Background information on #PyBay:**

[Pybay](https://pybay.com/) is offered by SF Python, a volunteer-run organization dedicated to building a stronger Python developer community. We do this by organizing fun events that encourage developers to meet like-minded folks, share what they know, and learn from each other. It’s hard work, but seeing people grow and make new friends and connections makes it worth it! We love hearing about companies making hires and migrating to better technologies as a result of our gatherings. We're also grateful to receive support for our work, financially and otherwise. We look forward to delighting you at PyBay and at other SF Python events throughout the year.

Looking forward to your response!

Cheers,
Raul Maldonad on behalf of the SF Python/PyBay Team

### CODE REVIEW

**Comment by jefftriplett** [comment]
@CloudChaoszero @Mariatta PyBay has been re-added twice to the CSV file both under "PyBay" and "PyBay 2019". That said, I think this PR should be closed.

**Comment by jonafato** [comment]
See #88 for where PyBay 2019 was originally added.

**Comment by Mariatta** [comment]
Ah! I didn't even notice that. Thanks @jefftriplett. I think the previous entry of PyBay is lacking information. It did not have venue info, CFP dates info, or sponsorship URL.

**Comment by CloudChaoszero** [comment]
@Mariatta @jefftriplett @jonafato Thanks! Just saw line #88, and decided to update that line as opposed to creating another one.

Again, thanks!

**Comment by Mariatta** [comment]
Thanks @CloudChaoszero. There are still two issues to be addressed:
- [ ]  keep date format as YYYY-MM-DD everywhere. In your PR the dates were all changed to MM/DD/YYYY
- [ ]  Resolve conflict

**Comment by CloudChaoszero** [comment]
@Mariatta  Done! Thanks!

File: 2019.csv [hunk 1]
```diff
@@ -20,7 +20,7 @@ SciPy,2019-07-08,2019-07-14,"Austin, Texas, United States of America",USA,,,,htt
 PyData London,2019-07-12,2019-07-14,"London, United Kingdom",GBR,The Tower Hotel,,,https://pydata.org/london2019,,
 PyOhio,2019-07-27,2019-07-28,"Columbus, OH",USA,The Ohio Union,2019-05-12,2019-05-12,https://www.pyohio.org/2019,https://www.pyohio.org/cfp,https://www.pyohio.org/2019/sponsorship
 PyCon AU,2019-08-02,2019-08-06,"Sydney, New South Wales, Australia",AUS,International Convention Centre,,,https://2019.pycon-au.org,,https://2019.pycon-au.org/sponsor/
-PyBay,2019-08-15,2019-08-18,"San Francisco, California, United States of America",USA,,,,https://pybay.com,,
+PyBay 2019,8-15-19,8-18-19,"San Francisco, California, United States of America",USA,Mission Bay Conference Center,5/12/19,5/12/19,https://pybay.com/,https://pybay2019cfp.hubb.me/,
```
Comment (Mariatta) [inline 2019.csv]:
```suggestion
PyBay,2019-08-15,2019-08-18,"San Francisco, California, United States of America",USA,Mission Bay Conference Center,2019-05-12,2019-05-12,https://pybay.com/,https://pybay2019cfp.hubb.me/,
```

File: 2019.csv [hunk 2]
```diff
@@ -33,4 +33,4 @@ PyGotham 2019,2019-10-04,2019-10-05,"New York City, New York, United States of A
 PyCon España,2019-10-04,2019-10-06,"Alicante, Spain",ESP,,,2019-05-10,https://2019.es.pycon.org,https://easychair.org/cfp/pycones2019,
 PyCon DE 2019,2019-10-09,2019-10-13,"Berlin, Germany",DEU,,,,https://de.pycon.org,,
 Python Brazil,2019-10-23,2019-10-28,"Ribeirão Preto, São Paulo, Brazil",BRA,Centro de Convenções Ribeirão Preto (CCRP),,,https://pythonbrasil.org.br,https://2019.pythonbrasil.org.br/submissao-de-palestras/,https://2019.pythonbrasil.org.br/assets/pdfs/sponsorship_plan.pdf
-North Bay Python 2019,2019-11-02,2019-11-03,"Petaluma, California, United States of America",USA,Mystic Theatre,,12,https://2019.northbaypython.org,,
+North Bay Python 2019,2019-11-02,2019-11-03,"Petaluma, California, United States of America",USA,Mystic Theatre,,12,https://2019.northbaypython.org,,
```
Comment (Mariatta) [inline 2019.csv]:
```suggestion
North Bay Python 2019,2019-11-02,2019-11-03,"Petaluma, California, United States of America",USA,Mystic Theatre,,12,https://2019.northbaypython.org,,

```

**Comment by CloudChaoszero** [comment]
@Mariatta  how about now? Pardon me for that.

**Comment by CloudChaoszero** [comment]
@Mariatta  QQ: When does [PyCon Calendar](https://pycon.org/) get updated? Thanks!

**Comment by jonafato** [comment]
That page does not currently use this repository as a data source, but it may in the future (this repository is not officially maintained by the PSF). There are separate instructions to add events to [that page](https://github.com/PyCon/pycon.org#adding-your-event) and the [events calendar](https://wiki.python.org/moin/PythonEventsCalendar#Submitting_an_Event).

**Comment by CloudChaoszero** [comment]
Thank you @jonafato !

**Pull request merged**